.github/workflows/product-release.yml

```
@@ -0,0 +1,296 @@
+
+  publish:
+    needs:
+      [plan, app-server-darwin, app-server-linux, validate-linux-arm, macos]
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    permissions:
+      contents: write
+    concurrency:
+      group: product-release-publish
+      cancel-in-progress: false
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)